diff --git a/node_modules/svelte-routing/index.d.ts b/node_modules/svelte-routing/index.d.ts
new file mode 100644
index 0000000..4f0715d
--- /dev/null
+++ b/node_modules/svelte-routing/index.d.ts
@@ -0,0 +1,9 @@
+declare module 'svelte-routing' {
+    import { SvelteComponent } from 'svelte'
+    export const Router: SvelteComponent
+    export const Route: SvelteComponent
+    export const Link: SvelteComponent
+    export const navigate: (to: string, { replace, state }: { replace?: boolean; state: object }) => void
+    export const link: (node: Element) => { destroy(): void }
+    export const links: (node: Element) => { destroy(): void }
+}
diff --git a/node_modules/svelte-routing/src/Link.svelte b/node_modules/svelte-routing/src/Link.svelte
index ff9eddf..d3b17ca 100644
--- a/node_modules/svelte-routing/src/Link.svelte
+++ b/node_modules/svelte-routing/src/Link.svelte
@@ -7,7 +7,7 @@
   export let to = "#";
   export let replace = false;
   export let state = {};
-  export let getProps = () => ({});
+  export let getProps = ({ location, href, isPartiallyCurrent, isCurrent }) => ({ class: '' });
 
   const { base } = getContext(ROUTER);
   const location = getContext(LOCATION);
